core/src/components/input/input.scss

```diff
@@ -259,15 +259,16 @@
 
 // Input Has focus
 // --------------------------------------------------
-// When the input has focus, then the input cover should be hidden
```

Comment:
The associated code for this comment was removed in https://github.com/ionic-team/ionic-framework/commit/d817cc3b30ae648730b063120b8d55b9ec09a53d#diff-25a4832c20c7342508f6e1921d7e0db8c6c7ef1b5053f643d665c8f45a84cd19R146.